mkdocs.yml

```diff
@@ -36,3 +57,10 @@ markdown_extensions:
       custom_checkbox: true
   - pymdownx.tilde
   - attr_list
+
+# Navigation
+nav:
+  - What is ShellHub?: index.md
+  - Introduction: introduction.md
+  - Getting started: getting-started.md
+  
```

Comment:
There is no newline at end of file. Likely previous line has a spurious space or something like that.